.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Please don't do unrelated changes.

Reply:
Removed it. The package-lock.json file is now generated. `package-lock.json` is not necessary for published package, that's why I added it.